### PR TITLE
Try and except direct messaging new users.

### PR DESCRIPTION
**Pull request by robbai**
In attempt to avoid [this issue](https://github.com/FelixRandle/Sheffield-Discord-Bot/issues/26).

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -62,23 +62,26 @@ async def on_member_join(member):
...
+            'we\'re all here to help each other!\n'
+            'Many thanks,\n'
+            'The Discord Server Admin Team\n\n'
+            'As a note, all messages that you send on the server are logged.\n'
+            'This is to help us in the case of messages that contain'
+            'offensive content and need to be reported.\n'
+            'If you would like your logged messages to be'
+            'removed for any reason, please contact <@247428233086238720>.'
+        )
+    except Exception:
```
Comment (WilliamWFLee) [inline main.py:83]: I think `discord.Forbidden` would be a more specific error to catch here

**Review by WilliamWFLee — CHANGES_REQUESTED** [review]
Thanks for your PR, Robbie.

I've left a comment for a minor improvement, but thanks anyway for contributing!

File: main.py (hunk 1 above)
Comment (robbai) [reply main.py:83]: Has `discord` been imported at this stage? If not I can just add `from discord import Forbidden` near the top of the file, or in the method.

Comment (WilliamWFLee) [reply main.py:83]: Oh yeah, can you put `import discord` in the import section at the top of the middle import section?

**Review by FelixRandle — APPROVED** [review]

**Comment by FelixRandle** [comment]
Looks good, cheers for taking the time to fix this.

**Pull request merged**